sdk/eventhub/azure-eventhubs/HISTORY.md

```diff
@@ -12,8 +12,9 @@
     - Renamed parameters in `get_partition_properties`: `partition` to `partition_id`.
 - Moved `consumer_group` parameter from `receive` method to the constructor of `EventHubConsumerClient`.
 - Renamed module `PartitionManager` to `CheckpointStore`.
-  
-
+- Removed class `EventPostition`.
+    - The `initial_event_position` parameter of the `receive` method accepts offset(`str`), sequence number(`int`), datetime (`datetime.datetime`) or `dict` of these types.
+    - The `initial_event_position_inclusive` parameter of the `receive` method accepts `bool` or `dict` indicating whether the given event position is inclusive or not.
```

Comment:
Also:
- PartitionContext no longer has attribute `owner_id`
- PartitionContext now has attribute `last_enqueued_event_properties` which is populated if `track...` is set to True.
- EventData no longer has attribute `last_enqueued_event_properties` - use this on PartitionContext instead.
- EventHubProducerClient.send has been renamed to `send_batch` which will only accept `EventDataBatch` object as input.
- EventHubProducerClient.create_batch now also takes the `partition_id` and `partition_key` as optional parameters (which are no longer specified at send).
- EventDataBatch.try_add has been renamed to EventDataBatch.add

Reply:
added, thanks for writing them down!